applications/FemToDemApplication/python_scripts/CouplingFemDem.py

```diff
@@ -842,9 +858,18 @@ def InitializeMMGvariables(self):
         ZeroVector3[0] = 0.0
         ZeroVector3[1] = 0.0
         ZeroVector3[2] = 0.0
+        ZeroVector6 = KratosMultiphysics.Vector(6)
+        ZeroVector6[0] = 0.0
+        ZeroVector6[1] = 0.0
+        ZeroVector6[2] = 0.0
+        ZeroVector6[3] = 0.0
+        ZeroVector6[4] = 0.0
+        ZeroVector6[5] = 0.0
 
         for node in self.FEM_Solution.main_model_part.Nodes:
```

Comment:
Use the VariableUtils()

Reply:
now I have removed this lines since are not necessary